tests/test011.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# respects max_plinks configuration
+#
+. `dirname $0`/include.sh
+
+[ $(uname) = "Linux" ] || err_exit "This test only runs on Linux"
+
+wait_for_clean_start
+
+nradios=8
+max_peers=4
+load_hwsim $nradios || err_exit "Failed to load mac80211-hwsim module."
+set_default_configs $nradios
+# set peer link limit
+for conf in ${CONFIGS[@]}; do
+    sed -i 's/meshid/max-plinks='$max_peers';&/' $conf
+done
+start_meshd $(get_hwsim_radios) || err_exit "Failed to start meshd-nl80211"
+
+wait_for_plinks $nradios
+
+# no radio should have more than max_peers peers
```

Comment:
I feel like here we'd expect to have *exactly* `max_peers` peers – unless there's a reason that's not the case, can we verify that too?

Reply:
Two reasons we might not:
 * wait_for_plinks only waits for a single peer, so there's a race condition
 * there's always the possibility we get a 5-clique with 4 peers each and then the other 3 have just 2 peers